docs/reference/sql/functions/approximate.md

```diff
@@ -0,0 +1,255 @@
+---
+keywords: [Approximate functions, approximate count distinct, approximate quantile, SQL functions]
+description: Lists and describes approximate functions available in GreptimeDB, including their usage and examples.
+---
+
+# Approximate Functions
+
+This page lists approximate functions in GreptimeDB, which are used for approximate data analysis.
+
+:::warning
+The following approximate functions is currently experimental and may change in future releases.
+:::
+
+## Approximate Count Distinct (HLL)
+
+The `hll` function is used to calculate the approximate count distinct of a set of values. It uses [HyperLogLog](https://algo.inria.fr/flajolet/Publications/FlFuGaMe07.pdf) (HLL) algorithm for efficient memory usage and speed. Three functions are provided for this purpose, described in following chapters:
```

Comment:
```suggestion
The HyperLogLog (HLL) algorithm is used to calculate the approximate count distinct of a set of values. It provides efficient memory usage and speed for this purpose. Three functions are provided to work with the HLL algorithm, described in following chapters:
```

I think this description is clearer because `hll` is just one function in the family implementing this algorithm, not all of them; emphasizing it alone can easily cause misunderstanding.